verifiable/did_doc.go

```diff
@@ -126,12 +126,25 @@ type StateInfo struct {
 
 // GistInfo representation state of gist root.
 type GistInfo struct {
-	Root                string `json:"root"`
-	ReplacedByRoot      string `json:"replacedByRoot"`
-	CreatedAtTimestamp  string `json:"createdAtTimestamp"`
-	ReplacedAtTimestamp string `json:"replacedAtTimestamp"`
-	CreatedAtBlock      string `json:"createdAtBlock"`
-	ReplacedAtBlock     string `json:"replacedAtBlock"`
+	Root                string     `json:"root"`
+	ReplacedByRoot      string     `json:"replacedByRoot"`
+	CreatedAtTimestamp  string     `json:"createdAtTimestamp"`
+	ReplacedAtTimestamp string     `json:"replacedAtTimestamp"`
+	CreatedAtBlock      string     `json:"createdAtBlock"`
+	ReplacedAtBlock     string     `json:"replacedAtBlock"`
+	Proof               *GistProof `json:"proof"`
```

Comment:
we need to make it optional in json, because teoretically some objects could be saved by someone. (put omitEmpty)